pkg/runner/step_context.go

```diff
@@ -220,6 +221,58 @@ func (sc *StepContext) setupAction(actionDir string, actionPath string) common.E
 		if os.IsNotExist(err) {
 			f, err = os.Open(filepath.Join(actionDir, actionPath, "action.yaml"))
 			if err != nil {
+				if sc.Step.With != nil {
+					if val, ok := sc.Step.With["args"]; ok {
+						trampoline := []byte(`
+const { spawnSync } = require('child_process')
```

Comment:
Rather than including this as a string within the Go code, how about using something like https://github.com/go-bindata/go-bindata to keep this file outside the go in a `.js` file?

Reply:
Can you be a lot more specific about how to do that?

Please assume I know absolutely nothing about golang...